lib/node_modules/@stdlib/math/base/special/factorial2/README.md

```diff
@@ -0,0 +1,141 @@
+# Double Factorial
+
+> [Double Factorial][double-factorial-function] function.
+
+
+<section class="intro">
+
+The [double factorial][double-factorial-function] function may be defined as the product
+
+<!-- <equation class="equation" label="eq:double_factorial_function" align="center" raw="n!! = \prod_{k=0}^m (n-2k) = n(n-2)(n-4)\ldots" alt="Double factorial function definition"> -->
+
+<div class="equation" align="center" data-raw-text="n!! = \prod_{k=0}^m (n-2k) = n(n-2)(n-4)\ldots" data-equation="eq:double_factorial_function">
+    <img src="" alt="Double factorial function definition">
+    <br>
+</div>
+
+<!-- </equation> -->
+
+or according to the recurrence relation
+
+<!-- <equation class="equation" label="eq:factorial_recurrence_relation" align="center" raw="n! = \begin{cases}1 &amp; \textrm{if } n = 0,\\(n-1)! \times n &amp; \textrm{if } n > 1\end{cases}" alt="Factorial function recurrence relation"> -->
+
+<div class="equation" align="center" data-raw-text="n! = \begin{cases}1 &amp; \textrm{if } n = 0,\\(n-2)! \times n &amp; \textrm{if } n > 1\end{cases}" data-equation="eq:factorial_recurrence_relation">
+    <img src="" alt="Double factorial function recurrence relation">
+    <br>
+</div>
+
+<!-- </equation> -->
+
+Following the convention for an [empty product][empty-product], in all definitions,
+
+<!-- <equation class="equation" label="eq:zero_double_factorial" align="center" raw="0!! = 1" alt="Zero double factorial"> -->
+
+<div class="equation" align="center" data-raw-text="0! = 1" data-equation="eq:zero_double_factorial">
+    <img src="" alt="Zero factorial">
+    <br>
+</div>
+
+<!-- </equation> -->
+
+The double factorial should not be confused with the factorial function iterated twice, written as (n!)! and not n!!
+
+The [double factorial][double-factorial-function] is implemented for integer values in terms of the [Factorial][@stdlib/math/base/special/factorial] function using the relation:
```

Comment:
Missing `[@stdlib/math/base/special/factorial]: https://github.com/stdlib-js/stdlib` in the link section at the end of the file.